### DISCUSSION TITLE
Poor outlet water temperature control - 3 H MT

### DISCUSSION THREAD

**Discussion by jf-wm** [raomin/ESPAltherma#345] [General]
I have a new Diakin Altherma 3H MT and operate it without a separate indoor or outdoor thermostat, all heating circuits are permanently open. The configuration is setup to control it using the outlet water temperature. I have noticed in the recorded data that the outlet temperature control works very poorly. The system does not reduce the heating output or compressor speed when the target output temperature is reached, but keeps increasing it. As can be clearly seen in the attached data, the actual outlet temperature is above the target value shortly after the start, but the system continues to increase the speed of the compressor and runs further and further away from the target value. Has anyone observed similar behaviour and found a solution? (The upper chart shows the temperature readings, dark blue outlet temp., light blue inlet temp. and purple target outlet temp.; the lower chart shows the compressor speed) 

![Bildschirmfoto 2023-11-04 um 20 36 43](https://github.com/raomin/ESPAltherma/assets/60154193/48098d01-f013-4a92-bf29-deb97e6e0b66)

**Comment by Arnold-n** [comment]
Daikins are famous for their short overshoots when they start (which frequently result in short-cycling), but this is something different. It seems to prioritize dT=3 over the AWT setpoint (AWT follows RWT), and will likely do so until it hits the  maximum overshoot  setting [9-04]. What is the flow measurement?

**Comment by jf-wm** [comment]
Thanks for your quick answers. I also noticed that dT was keep constant, but I don't understand why. The maximum overshoot is set to 2°C as the system had very short cycles using 1°C. The flow is modulated slightly over the time period:

![Bildschirmfoto 2023-11-04 um 21 50 03](https://github.com/raomin/ESPAltherma/assets/60154193/66269aa2-18cc-4d47-b41b-a5b9c21b5f52)

**Comment by Eva-Husum** [comment]
The leaving water temperature that you set as “Soll Vorlauf” is the temperature of the water which is leaving the indoor unit and goes to your heating circuit. Are you sure your “Vorlauf Temp1” is a measurement of that value? Or is it maybe the outlet water temperature of the outdoor heat exchanger? I also have a 3H MT and so far I have not found any way of reading the actual outlet temperature of the indoor unit through ESPAltherma. I can only read it from the display at the indoor unit itself… Which register address are you using for reading “Vorlauf Temp1”?

**Reply by jf-wm** [reply]
"Soll Vorlauf" is the target outlet temperature not the actual temperature, I read register (0x60,9,105,2,1) for this value. "Vorlauf Temp1" is the actual outlet temperature after the internal resistive heater in the inside unit, however that value is only marginally (within sensor accuracy) different from the temperature reading of the outlet temperature after the heat exchanger in the outdoor unit. "Vorlauf Temp1" refers to the sensor R2T, register (0x61,4,105,2,1), the outlet temperature sensor in the outdoor unit is R1T, register (0x61,2,105,2,1).

**Reply by Eva-Husum** [reply]
![IMG_6213](https://github.com/raomin/ESPAltherma/assets/135121792/90105590-a729-4996-924c-e76390df92d4)
Ok, that’s understood. However, neither R1T nor R2T are the actual outlet temperature (Vorlauftemperatur). You would need to read R7T for the actual outlet temperature after the mixing valve. Please see the picture attached. 
Unfortunately I haven’t found a way to read R7T from ESPAltherma…
(Off topic: don’t worry, I’m not normally operating my heatpump with such high temperatures as shown in the picture. I have only switched it to SmartGrid Forced On in order to make the differences in temperatures clearly visible.)

**Reply by markoweb** [reply]
@Eva-Husum How could be reached the screen from your last post?

**Reply by Eva-Husum** [reply]
Hi Markoweb, in the German user interface it is in „Information“ and then „Rohrleitungsplan“
